frappe/website/doctype/blog_post/blog_post.py

```diff
@@ -101,7 +101,7 @@ def load_comments(self, context):
 				context.comment_text = _('{0} comments').format(len(context.comment_list))
 
 	def set_read_time(self):
-		content = self.content or self.content_html
+		content = (self.content or self.content_html) or ''
```

Comment:
@abhishekbalam Remove the brackets, they are unnecessary
`content = self.content or self.content_html or ''`

Reply:
@hrwX 
Sure, 
but I cant push changes to this, so will close and reopen this pr.

Reply:
Sure